rclpy/test/test_node.py

```diff
@@ -838,6 +842,103 @@ def test_node_set_parameters_rejection(self):
         self.assertIsInstance(result[0], SetParametersResult)
         self.assertFalse(result[0].successful)
 
+    def test_node_set_paramters_rejection_list(self):
+        # Declare a new paramters and set a list of callbacks so that it's rejected when set.
+        reject_list_parameter_tuple = [
+            (
+                'reject1', True, ParameterDescriptor()
+            ),
+            (
+                'non_reject', True, ParameterDescriptor()
+            ),
+            (
+                'non_reject', True, ParameterDescriptor()
```

Comment:
Please make these one line.

```py
('non_reject', True, ParameterDescriptor())
```

Reply:
> > Now I should rebase my current branch ros2/master to get the update my repo isnt it?
> 
> I'm confused what you mean?

I mean to say do I have sync my repo with that of ros2/master

Reply:
@allenh1 sir is anything more I need to fix?